index.html

```diff
@@ -395,6 +395,28 @@ <h2>Convert a <var>name</var> to a <var>timestamp</var></h2>
         The <a data-cite="NAVIGATION-TIMING#performancetiming">PerformanceTiming</a> interface was defined in [[NAVIGATION-TIMING]] and is now considered obsolete. The use of names from the <a data-cite="NAVIGATION-TIMING#performancetiming">PerformanceTiming</a> interface is supported to remain backwards compatible, but there are no plans to extend this functionality to names in the <a data-cite="NAVIGATION-TIMING-2#dom-performancenavigationtiming">PerformanceNavigationTiming</a> interface defined in [[NAVIGATION-TIMING-2]] (or other interfaces) in the future.
       </p>
     </section>
+  </section><section id="recommended-mark-names" class="informative">
+    <h2>Recommended mark names</h2>
+    <p>Developers are encouraged to use the following recommended mark names to
```

Comment:
Maybe "recommended mark names" here should be a <dfn>?

Reply:
I think you were trying to write an HTML tag and github ate everything between the angle brackets?

Should be what?

Reply:
I added `<dfn>` around these, but now I get an error:

> Found definition for "Recommended mark names", but nothing links to it. This is usually a spec bug!

Reply:
Use `data-export` to declare that you expect the definition to be used from outside this spec. I actually wouldn't `<dfn>` this header, but `<dfn>`'ing the terms is good.

Reply:
(see suggestion above...)